Formula/embulk.rb

```diff
@@ -1,8 +1,8 @@
 class Embulk < Formula
   desc "Data transfer between various databases, file formats and services"
   homepage "https://www.embulk.org/"
-  url "https://bintray.com/artifact/download/embulk/maven/embulk-0.9.23.jar"
-  sha256 "153977fad482bf52100dd96f47e897c87b48de4fb13bccd6b3101475d3a5ebb9"
+  url "https://jcenter.bintray.com/org/embulk/embulk-core/0.10.1/embulk-core-0.10.1.jar"
```

Comment:
This does not exist in bintry (searched in here, https://bintray.com/beta/#/embulk/maven/embulk?tab=files).